index.js

```diff
@@ -12,7 +12,7 @@ function Chalk(options) {
 }
 
 // use bright blue on Windows as the normal blue color is illegible
-if (process.platform === 'win32') {
+if (process.platform === 'win32' && !(/^xterm/i).test(process.env.TERM || 'dumb')) {
```

Comment:
Please copy paste what I commented, the regex doesn't need parens and `dumb` check is invalid.